index.html

```diff
@@ -9,11 +9,66 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+<body>
+    <header>
+       
+    <!--Navigation--> 
+    <nav>    
+        <a href="#" class="logo"> 
+           <img src="img/karma-logo.svg" alt="logo">
+        </a>     
+        <a href="">Meet Karma</a>
+        <a href="">How it works</a>
+        <a href="">Store</a>
+        <a href="">Blog</a>
+        <a href="">Help</a>
+        <a href="">Log in</a>
```

Comment:
```suggestion
        <ul>
            <li><a href="">Meet Karma</a></li>
            <li><a href="">How it works</a></li>
            <li><a href="">Store</a></li>
            <li><a href="">Blog</a></li>
            <li><a href="">Help</a></li>
            <li><a href="">Log in</a></li>    
        </ul>
```
In a `nav`, there's usually a list of links, and because it's a list we usually put them in an HTML list. We can then use CSS to remove the bullet points and change the list to inline boxes